src/background.ts

```diff
@@ -14,18 +13,23 @@ function getOauth2Client(origin: string) {
     ? env.DEV_OAUTH2_CLIENT_ID
     : env.OAUTH2_CLIENT_ID;
   const subjectId = isDev(origin)
-    ? env.DEV_OAUTH2_SUBJECT_ID
-    : env.OAUTH2_SUBJECT_ID;
+    ? null
+    : env.OAUTH2_SUBJECT_ID[env.renderBackend];
+  const subjectName = isDev(origin)
+    ? env.DEV_OAUTH2_SUBJECT_NAME[env.renderBackend]
+    : null;
```

Comment:
It seems like the pattern used to structure these constants doesn't work anymore -- can we think of a cleaner way then assigning nulls?

Reply:
Decided to switch to using names instead of hardcoding ID of the subjects for both dev and prod. It's one more request, but since the result is cached, I don't think it maters much...